pkg/converter/convert_unix.go

```diff
@@ -712,41 +726,61 @@ func MergeLayers(ctx context.Context, cs content.Store, descs []ocispec.Descript
 	buffer := bufPool.Get().(*[]byte)
 	defer bufPool.Put(buffer)
 	if _, err := io.CopyBuffer(compressed, pr, *buffer); err != nil {
-		return nil, errors.Wrapf(err, "copy bootstrap targz into content store")
+		return nil, nil, errors.Wrapf(err, "copy bootstrap targz into content store")
 	}
 	if err := gw.Close(); err != nil {
-		return nil, errors.Wrap(err, "close gzip writer")
+		return nil, nil, errors.Wrap(err, "close gzip writer")
 	}
 
 	compressedDgst := cw.Digest()
 	if err := cw.Commit(ctx, 0, compressedDgst, content.WithLabels(map[string]string{
 		LayerAnnotationUncompressed: uncompressedDgst.Digest().String(),
 	})); err != nil {
 		if !errdefs.IsAlreadyExists(err) {
-			return nil, errors.Wrap(err, "commit to content store")
+			return nil, nil, errors.Wrap(err, "commit to content store")
 		}
 	}
 	if err := cw.Close(); err != nil {
-		return nil, errors.Wrap(err, "close content store writer")
+		return nil, nil, errors.Wrap(err, "close content store writer")
 	}
 
-	info, err := cs.Info(ctx, compressedDgst)
+	bootstrapInfo, err := cs.Info(ctx, compressedDgst)
 	if err != nil {
-		return nil, errors.Wrap(err, "get info from content store")
+		return nil, nil, errors.Wrap(err, "get info from content store")
+	}
+
+	blobDigests := <-blobDigestChan
+	blobDescs := []ocispec.Descriptor{}
+	blobIDs := []string{}
+	for _, blobDigest := range blobDigests {
```

Comment:
If `blobDigests` is empty or nil, which means another processing goroutine fails . Should it abort the following handle?

Reply:
Will abort at L728: `if _, err := io.CopyBuffer(compressed, pr, *buffer); err != nil {`